### PR TITLE
Add util function to ensure a trailing / on publicUrl

### PR DESCRIPTION
**Pull request by Huanzhang89**

### CODE REVIEW

File: src/utils/url.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 // prefix helper
-const prefixUrl = (url, publicUrl) =>
+export const prefixUrl = (url, publicUrl) =>
   url.startsWith('data:') ? url : publicUrl + url
 
-export { prefixUrl }
+// trailing slash helper
+export const ensureSlash = path => {
+  return path.endsWith('/') ? `${path}/` : path
```
Comment (bpierre) [inline src/utils/url.js]: Shouldn’t it be the other way?

Comment (bpierre) [inline src/utils/url.js]: I wonder if we could name it something like `ensureFinalSlash`, `ensureTrailingSlash` or just `trailingSlash`, to make it clear that it is added at the end of the string?

Comment (Huanzhang89) [reply src/utils/url.js]: oh wow mistakes were made

Comment (Huanzhang89) [reply src/utils/url.js]: Yeah agreed, will update the name

**Pull request merged**